src/content-cli-import.ts

```diff
@@ -0,0 +1,44 @@
+import {CommanderStatic} from "commander";
+import {PackageCommand} from "./commands/package.command";
+import {logger} from "./util/logger";
+import {contextService} from "./services/context.service";
+import * as commander from "commander";
+
+export class Import {
+
+    public static packages(program: CommanderStatic): CommanderStatic {
+        program
+            .command("packages")
+            .description("Command to import all given packages")
+            .option("-p, --profile <profile>", "Profile which you want to use to list packages")
+            .option("--spaceMapping <spaceMapping...>", "Mapping on which space packages will be created", "[]")
+            .option("--exportedDatapoolsFile <exportedDatapoolsFile>", "Exported datapool file (relative path)", "")
```

Comment:
Let's discuss this a bit more, as the data pools should probably be included in the same export file. @frkntrcnn FYI